### ISSUE TITLE
Keeping the Ubuntu APT repo up to date

### ISSUE THREAD

**Issue by CleanMachine1** [psibi/tldr-hs#37]
The APT repo has tldr within, as you probably know.
However it is incredibly out of date, a version from over 2 years ago.
Do you have anyway to update it to most recent or am I contacting the wrong place

**Comment by psibi** [comment]
Unfortunately, I don't maintain the package in Ubuntu.

One easy way to get binaries is via the downloads available here: https://github.com/psibi/tldr-hs/releases/tag/v0.9.0

**Comment by bl-ue** [comment]
(for future reference: https://github.com/tldr-pages/tldr/issues/5937)

**Comment by psibi** [comment]
> Do you know who may be able to help me,

Unfortunately no... Probably try their mailing list ?

But I think the recent version of Ubuntu ships with a relatively new one: http://manpages.ubuntu.com/manpages/impish/man1/tldr-hs.1.html

**Comment by CleanMachine1** [comment]
Thank you, please leave this issue open incase anyone else has anything to add

**Comment by patricedenis** [comment]
yeah, this is a good news, in fact.
Congrats !! :clap:

**Comment by patricedenis** [comment]
Hi @psibi 
I was trying to update the debian tldr package when I met several errors but this one I can't understand.
May be you could help me?

`. /usr/share/haskell-devscripts/Dh_Haskell.sh && \
install_prof_recipe "libghc-tldr-prof"
Running cd debian/tmp-inst-ghc
Running mkdir -p usr/lib/haskell-packages/ghc/lib
Running find usr/lib/haskell-packages/ghc/lib/ \! \( \! -name \*_p.a \! -name \*.p_hi \) -exe
c install -Dm 644 \{\} ../libghc-tldr-prof/\{\} \;
Running dh_haskell_provides -plibghc-tldr-prof 
Running dh_haskell_depends -plibghc-tldr-prof
ghc-pkg: cannot parse 'semigroups-0.19.1-LKdktuRRdRZ1yx9gBXzSbK' as a package identifier (exact): "<eitherParsec>" (line 1, column 41):
unexpected all digits or a dot in a portion of package name
expecting "-"
make: *** [/usr/share/cdbs/1/class/hlibrary.mk:212: install/libghc-tldr-prof] Error 1
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
I: copying local configuration
E: Failed autobuilding of package`

as I understand, the compiler should produce a libghc-tldr-prof target but it fails.
what I don't understand is why it referes to the 'semigroups-0.19.1-LKdktuRRdRZ1yx9gBXzSbK' complete ID where I mentionned in the build dependencies only the name of the modules added after tage 0.6.4.

By the way is it true that the project depends from these new modules ?
libghc-optparse-applicative-prof,
libghc-zip-archive-prof,
libghc-http-conduit-prof
libghc-optparse-applicative-prof,
libghc-zip-archive-prof,
libghc-http-conduit-prof

I added them in the dependencies to build to debian package.

An other question is : has I saw in the commit history, ou updated the associated library from v0.8.0.
should I build other targets than the following ones in my package?
libghc-tldr-dev, libghc-tldr-prof, libghc-tldr-doc, libghc-tldr

Thanks for your time.

**Comment by psibi** [comment]
> what I don't understand is why it refers to the 'semigroups-0.19.1-LKdktuRRdRZ1yx9gBXzSbK' 

Probably because `semigroups` is one of the transitive dependency.

> By the way is it true that the project depends from these new modules ?

Yeah, that sounds correct.

> An other question is : I saw in the commit history that you splited the library. This is from release v0.8.0.

Yeah, that is intentional.

> should I build other targets than the following ones in my package?
> libghc-tldr-dev, libghc-tldr-prof, libghc-tldr-doc, libghc-tldr

I think the only major thing important is `libghc-tldr` which likely contains the binary. The other ones aren't that important, but I don't know what's Debian policy on these things.

**Comment by patricedenis** [comment]
> Probably because semigroups is one of the transitive dependency.

yeah I included the semigroups lib into the dependencies but at compilation time I've got an error saying that the number should not appear where I did not mention any number to this lib

` unexpected all digits or a dot in a portion of package name expecting

> I think the only major thing important is libghc-tldr which likely contains the binary. The other ones aren't that important, but I don't know what's Debian policy on these things.

As I saw, all the libraries were built on the previous version, that's why I tried to do the same here.
